cmd/rpcdaemon/graphql/graph/helpers.go

```diff
@@ -67,15 +68,12 @@ func convertDataToIntP(abstractMap map[string]interface{}, field string) *int {
 
 	switch v := abstractMap[field].(type) {
 	case hexutil2.Uint64:
+	case *hexutil2.Uint:
 		resultUint, err := hexutil2.DecodeUint64(v.String())
 		if err != nil {
 			result = 0
-		} else {
-			result = int(resultUint)
-		}
-	case hexutil2.Uint:
-		resultUint, err := hexutil2.DecodeUint64(v.String())
-		if err != nil {
+		} else if resultUint > uint64(math.MaxInt) {
```

Comment:
Can we not use `hexutil2.DecodeInt64` here?  This will support the `*int` return and will error on an overflow.

Reply:
Frankly I dont understand what you mean here. Could you please explain?
We are handling Uint type in switch, which is 32 or 64 bits. So I am decoding it to uint64. since this is unsigned in switch as type, does it make sense to expect negative numbers here? I think it is just the interface that wants int as return value but it will never be negative.

The initial implementation did the same and I find it valid:
```
      case hexutil2.Uint64:
                     resultUint, err := hexutil2.DecodeUint64(v.String())
```